charts/tekton-pipelines/templates/kaniko/tasks/kaniko.yaml

```diff
@@ -69,6 +84,11 @@ spec:
         #!/bin/bash
         aws ecr get-login-password --region $AWS_REGION > $(results.password.path)
 
+    # insert customized steps as defined in the values.yaml of the helm chart
```

Comment:
this has to be a dedicated step, not part of the `authenticate` step

Reply:
and also - do we need this code to be part of the docker image? If not - let's move after  the build step

Reply:
that is a dedicated step, it is not located in `authenticate`, please take a look on it here - https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/feature/add-frontend-sentry-sourcemaps-load/charts/tekton-pipelines/values.yaml#L364

Reply:
@dmitry-mightydevops removed this (described why in PR description in `kaniko-frontend` section)

Reply:
Ok have you see this alarm?
![image](https://github.com/saritasa-nest/saritasa-devops-helm-charts/assets/70822710/7a598411-f361-41e8-9727-7333e0c17461)

Reply:
fixed